lib/Migration/Version10900Date20221020074643.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * @copyright Copyright (c) 2022 Thanos kamber <thanos.kamber@gmail.com>
+ *
+ * @author Thanos kamber <thanos.kamber@gmail.com>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace OCA\Deck\Migration;
+
+use Closure;
+use OCP\DB\ISchemaWrapper;
+use OCP\Migration\IOutput;
+use OCP\Migration\SimpleMigrationStep;
+
+/**
+ * Auto-generated migration step: Please modify to your needs!
+ */
+class Version10900Date20221020074643 extends SimpleMigrationStep {
+	/**
+	 * @param IOutput $output
+	 * @param Closure $schemaClosure
+	 * @param array $options
+	 * @return null|ISchemaWrapper
+	 */
+	public function changeSchema(IOutput $output, Closure $schemaClosure, array $options): ?ISchemaWrapper {
+		/** @var ISchemaWrapper $schema */
+		$schema = $schemaClosure();
+
+		$table = $schema->getTable('deck_cards');
+		if (!$table->hasColumn('done')) {
+			$table->addColumn('done', 'boolean', [
+				'default' => false,
+				'notnull' => true,
+			]);
+		}
+
+		return $schema;
```

Comment:
Best practice would be to only return the schema if there was a change:

```suggestion
		$table = $schema->getTable('deck_cards');
		if (!$table->hasColumn('done')) {
			$table->addColumn('done', 'boolean', [
				'default' => false,
				'notnull' => true,
			]);
			
			return $schema;
		}
		
		return null;
```

Reply:
One addition that we would need is to also change the type in the original migration, so that new installations can skip the later one.